src/tool/xmeta/cpp/models/namespace_model.cpp

```diff
@@ -1,59 +1,69 @@
 #include "namespace_model.h"
 
-namespace xlang
+namespace xlang::xmeta
 {
-    namespace xmeta
-    {
 #pragma region namespace_body_model
-        bool namespace_body_model::member_id_exists(const std::string &member_id)
+    bool namespace_body_model::member_id_exists(std::string_view const& member_id)
+    {
+        if (classes.find(member_id) != classes.end())
         {
-            if (classes.find(member_id) != classes.end())
-                return true;
-
-            if (structs.find(member_id) != structs.end())
-                return true;
-
-            if (interfaces.find(member_id) != interfaces.end())
-                return true;
+            return true;
+        }
 
-            return false;
+        if (structs.find(member_id) != structs.end())
+        {
+            return true;
         }
 
-        std::string namespace_body_model::get_full_namespace_name()
+        if (interfaces.find(member_id) != interfaces.end())
         {
-            return containing_namespace->get_full_namespace_name();
+            return true;
         }
+
+        return false;
+    }
+
+    std::string namespace_body_model::get_full_namespace_name()
+    {
+        return containing_namespace->get_full_namespace_name();
+    }
 #pragma endregion
 
 
 #pragma region namespace_model
-        namespace_model::namespace_model(const std::string &id, const size_t &decl_line, const std::shared_ptr<namespace_model> &pn) :
-            base_model{ id, decl_line },
-            parent_namespace{ pn }
-        { }
+    namespace_model::namespace_model(std::string_view const& id, size_t decl_line, std::shared_ptr<namespace_model> const& parent) :
+        base_model{ id, decl_line },
+        parent_namespace{ parent }
+    { }
 
-        bool namespace_model::member_id_exists(const std::string &member_id)
+    bool namespace_model::member_id_exists(std::string_view const& member_id)
+    {
+        for (auto ns_body : ns_bodies)
         {
-            for (auto ns_body : ns_bodies)
+            if (ns_body->member_id_exists(member_id))
             {
-                if (ns_body->member_id_exists(member_id))
-                {
-                    return true;
-                }
-            }
-
-            if (child_namespaces.find(member_id) != child_namespaces.end())
                 return true;
+            }
+        }
 
-            return false;
+        if (child_namespaces.find(member_id) != child_namespaces.end())
```

Comment:
nit: Boolean zen, just use a return statement instead of "if true return true else return false" 

return child_namespaces.find(member_id) != child_namespaces.end();